addons/project/static/src/views/project_calendar/project_calendar_model.js

```diff
@@ -0,0 +1,17 @@
+/** @odoo-module */
+
+import { CalendarModel } from '@web/views/calendar/calendar_model';
+
+export class ProjectCalendarModel extends CalendarModel {
+    /**
+     * @override
+     * @returns {String}
+     */
+    defaultFilterLabel() {
+        this.isCheckProject = Object.keys(this.meta.filtersInfo).includes('project_id')
```

Comment:
I am wondering if it is not equivalent to:
```suggestion
        this.isCheckProject = 'project_id' in this.meta.filtersInfo
```